Target-MVD-Mathias/Networking/Services/UserAPI.swift

```diff
@@ -137,6 +136,25 @@ class UserAPI {
     }
   }
   
+  class func resetPassword(_ previousPassword: String, newPassword: String, success: @escaping () -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let url = usersUrl + "\(UserDataManager.getUserId())" + "/password/change"
```

Comment:
You can avoid that last concat by doing `let url = usersUrl + "\(UserDataManager.getUserId())/password/change"`

Reply:
Copy paste issues 🤦‍♂️